variables.tf

```diff
@@ -4,6 +4,8 @@ variable "log_analytics_workspace" {
     name                = string
     resource_group_name = string
     location            = string
+    workspace_id        = string
+    primary_shared_key  = string
```

Comment:
I would put the primary_shared_key in a separate variable.

1. It's a secret and needs to be marked as sensitive
2. The CAF output doesn't actually include this output (because it's a secret)